spacy_legacy/architectures/parser.py

```diff
@@ -17,35 +18,15 @@ def TransitionBasedParser_v1(
     use_upper: bool = True,
     nO: Optional[int] = None,
 ) -> Model:
-
-    chain = registry.get("layers", "chain.v1")
-    list2array = registry.get("layers", "list2array.v1")
-    Linear = registry.get("layers", "Linear.v1")
-    TransitionModel = registry.get("layers", "spacy.TransitionModel.v1")
-    PrecomputableAffine = registry.get("layers", "spacy.PrecomputableAffine.v1")
-
-    if state_type == "parser":
-        nr_feature_tokens = 13 if extra_state_tokens else 8
-    elif state_type == "ner":
-        nr_feature_tokens = 6 if extra_state_tokens else 3
-    else:
-        raise ValueError(Errors.E917.format(value=state_type))
-    t2v_width = tok2vec.get_dim("nO") if tok2vec.has_dim("nO") else None
-    tok2vec = chain(tok2vec, list2array(), Linear(hidden_width, t2v_width))
-    tok2vec.set_dim("nO", hidden_width)
-    lower = PrecomputableAffine(
-        nO=hidden_width if use_upper else nO,
-        nF=nr_feature_tokens,
-        nI=tok2vec.get_dim("nO"),
-        nP=maxout_pieces,
+    # TODO: Add back use_upper once it is implemented in the refactored parser.
```

Comment:
Would the lack of `use_upper` break any existing configs that use the `v1` parser?

Reply:
No, but this is also planned to be restored before the `v4` release.